### DISCUSSION TITLE
List of supported apps?

### DISCUSSION THREAD

**Discussion by nichu42** [NexovaDev/UpdateHub#301] [Q&A] [answered]
Is there a way to see the list of supported apps that UH supports?
This would be essential for me since I have to take care of unsupported apps myself.

**Comment by onyx-nxt** [comment, marked as answer]
Any app in this repository (https://github.com/microsoft/winget-pkgs) is supported.

**Reply by nichu42** [reply]
Thanks. In the future, it would be cool to have a list inside the UH app that shows supported apps that are installed, Much like it used to be in SUMo (many people, including me, are coming from there).
